src/shared/services/activity/activity.service.ts

```diff
@@ -66,22 +75,22 @@ export class ActivityService {
     const options = this.getOptions();
     const url = `${this.activityUrl}/cases/${caseId.join(',')}/activity`;
     return this.http
-      .get(url, options, false)
+      .get(url, options, false, ActivityService.handleHttpError)
       .map(response => response);
   }
 
-  public postActivity(caseId: string, activityType: String): Observable<Activity[]> {
+  public postActivity(caseId: string, activity: String): Observable<Activity[]> {
     const options = this.getOptions();
     const url = `${this.activityUrl}/cases/${caseId}/activity`;
-    let body = { activity: activityType};
+    let body = { activity };
     return this.http
       .post(url, body, options, false)
       .map(response => response);
```

Comment:
I don't understand why we need a `map` here? I have seen it many times on this project does it solve some kind of bug or something. If it does we need a comment.

Reply:
An excellent point. This code was already in place and I didn't really assess it in a huge amount of detail as it was working fine... but you're right - the `map` is a no-op and so not needed at all. I'll change it.